test/integration/features/basic.feature

```diff
@@ -147,6 +147,12 @@ Feature: Basic
      """
      exposed
      """
+
+  Scenario: User can access ruby-ex routing url
+   Given Minishift has state "Running"
+     When status code of HTTP request to "Routing" is equal to "200"
+     Then body of HTTP request to "Routing" contains "ok"
```

Comment:
"body of HTTP request to (.*) contains", is a generic step, which allows you to make a HTTP request and make an assertion about the content of the response. "ok" makes sense for the health check earlier on this feature file, since the health check URL will literally just return "ok". Your test only works since the demo webpage contains accidentally the substring "ok".